src/components/tl/sharp/tl_sharp_team.c

```diff
@@ -36,7 +37,10 @@ UCC_CLASS_INIT_FUNC(ucc_tl_sharp_team_t, ucc_base_context_t *tl_context,
     set.map    = UCC_TL_TEAM_MAP(self);
 
     if (UCC_TL_SHARP_TEAM_LIB(self)->cfg.use_internal_oob) {
-        self->oob_ctx.subset = set;
+        ucc_ep_map_create_nested(&UCC_TL_CORE_TEAM(self)->ctx_map,
```

Comment:
The map needs to be cleaned in the error path of the current function

Reply:
fixed